### PR TITLE
Update vercel.mdx

### PR DESCRIPTION
**Pull request by derekpcollins**
#### What kind of changes does this PR include?

- New or updated content

#### Description

I added a tip for making sure your project is using Node 16.12.0 or later if you're upgrading to Astro v2.0.

### CODE REVIEW

**Comment by Jutanium** [comment]
Thanks for this! I moved the disclaimer to its own section of **How to deploy**, since it applies both to static and SSR builds, and since we default to sections over tips. If this looks good to you, we'll go ahead and merge!

**Comment by derekpcollins** [comment]
This looks great and I've noted that sections are preferred over tips for future reference (I wasn't sure what the best way to display this information would be).

I'd suggest rephrasing the linked text to read something like:

You can [define the Node.js version](https://vercel.com/changelog/node-js-version-now-customizable-in-the-project-settings) used during the Build Step and Serverless Functions from the General page under Project Settings.

The Node.js version affects the Build Step and Serverless Functions, but the setting itself is found in General > Settings, if that makes sense.

File: src/pages/en/guides/deploy/vercel.mdx [hunk 1]
```diff
@@ -82,3 +82,7 @@ After your project has been imported and deployed, all subsequent pushes to bran
 You can use `vercel.json` to override the default behavior of Vercel and to configure additional settings. For example, you may wish to attach headers to HTTP responses from your Deployments. 
 
 📚 Learn more about [Vercel’s project configuration](https://vercel.com/docs/project-configuration).
+
+### Upgrading to Astro 2.0
+
+Astro v2.0 drops support for Node 14, so make sure your project is using **Node `16.12.0` or later**. You can [define the Node.js version](https://vercel.com/changelog/node-js-version-now-customizable-in-the-project-settings) used during the Build Step and Serverless Functions from the General page under Project Settings.
```
Comment (Jutanium) [inline src/pages/en/guides/deploy/vercel.mdx:88]:
```suggestion
Astro v2.0 drops support for Node 14, so make sure your project is using **Node `16.12.0` or later**. You can [define the Node.js version](https://vercel.com/changelog/node-js-version-now-customizable-in-the-project-settings) used during the Build Step and in Serverless Functions from the General page under Project Settings.
```
Final nit and we're good to go!

Comment (derekpcollins) [reply src/pages/en/guides/deploy/vercel.mdx:88]: Looks great, thank you!

**Pull request merged**